STACpopulator/implementations/CMIP6_UofT/collection_config.yml

```diff
@@ -4,4 +4,17 @@ description: Coupled Model Intercomparison Project phase 6
 keywords: ['CMIP', 'CMIP6', 'WCRP', 'Climate Change']
 license: "CC-BY-4.0"
 spatialextent: [-180, -90, 180, 90]
-temporalextent: ['1850-01-01', null]
\ No newline at end of file
+temporalextent: ['1850-01-01', '2500-01-01']
+
+links:
+  homepage:
+      rel: about
+      title : Project homepage
+      target : https://wcrp-cmip.org/cmip-phase-6-cmip6/
+      media_type: text/html
+  license:
+      rel: license  
+      title : License
+      target : https://raw.githubusercontent.com/WCRP-CMIP/CMIP6_CVs/main/LICENSE-CC-BY
+      media_type: text/plain
```

Comment:
Why use a dict instead of a list if the link name is dropped anyway?

Reply:
That's a good idea. Done.